app/views/mediators.html

```diff
@@ -8,6 +8,9 @@
         <div class="panel-heading">
           <div class="panel-title">
             <h2><i class="glyphicon glyphicon-link"></i>&nbsp;&nbsp;Mediators</h2>
+            <p>
+              Mediators are add on services that run separatly from the OpenHIM. They register themselves with the OpenHIM and once that is done they will be displayed here and their configuration details may be modified. Also, if a mediator is registered it will allow you to easy add routes for it in the channel configuration.
```

Comment:
_Mediators are add**-**on services that run separat**a**ly from the OpenHIM. They register themselves with the OpenHIM and once that is done they will be displayed here and their configuration details may be modified. Also, if a mediator is registered it will allow you to eas**il**y add routes for it in the channel configuration._